src/django/api/tests/test_wage_indicator_provider.py

```diff
@@ -0,0 +1,238 @@
+        try:
+            self.partner_field = PartnerField.objects \
+                .get_all_including_inactive() \
+                .get(name='wage_indicator')
+        except PartnerField.DoesNotExist:
+            self.partner_field = PartnerField.objects.create(
+                name='wage_indicator',
+                type=PartnerField.OBJECT,
+                label='Wage Indicator',
+                system_field=True,
+                active=True
+            )
+
+        # Clear all existing contributors and set only this contributor
+        # to ensure test isolation from previous tests.
+        self.partner_field.contributor_set.clear()
+        self.contributor.partner_fields.add(self.partner_field)
+        self.partner_field.refresh_from_db()
+
+        # Create test production location with valid country code.
+        self.test_country_code = 'GB'  # Use GB for testing.
+        facility_list = FacilityList.objects.create(
+            header='header',
+            file_name='test',
+            name='Test List'
+        )
+
+        source = Source.objects.create(
+            facility_list=facility_list,
+            source_type=Source.LIST,
+            is_active=True,
+            is_public=True,
+            contributor=self.contributor,
+        )
+
+        list_item = FacilityListItem.objects.create(
+            name='Test Facility',
+            address='123 Test St',
+            country_code=self.test_country_code,
+            sector=['Apparel'],
+            row_index=0,
+            geocoded_point=Point(0, 0),
+            status=FacilityListItem.CONFIRMED_MATCH,
+            source=source,
+        )
+
+        self.facility = Facility.objects.create(
+            name='Test Facility',
+            address='123 Test St',
+            country_code=self.test_country_code,
+            location=Point(0, 0),
+            created_from=list_item,
+        )
+
+        FacilityMatch.objects.create(
+            status=FacilityMatch.CONFIRMED,
+            facility=self.facility,
+            results='',
+            facility_list_item=list_item,
+        )
+
+        list_item.facility = self.facility
+        list_item.save()
+
+        # Clean up any existing wage data for test country.
+        WageIndicatorCountryData.objects.filter(
+            country_code=self.test_country_code
+        ).delete()
+
+        # Create wage indicator data for test country.
+        self.wage_data = WageIndicatorCountryData.objects.create(
+            country_code=self.test_country_code,
+            living_wage_link_national='https://test.example.com/living',
+            minimum_wage_link_english='https://test.example.com/min-en',
+            minimum_wage_link_national='https://test.example.com/min-nat',
+        )
+
+        self.provider = WageIndicatorProvider()
+
+    def tearDown(self):
+        '''Clean up test data but preserve system partner field.'''
+        # Remove contributor assignments.
```

Comment:
According to best practices the comments should answer the question `Why` something is being done a certain way, this seems like it's just describing `What` is being done.